### ISSUE TITLE
How may we incorporate a new loss function in deeponet?

### ISSUE THREAD

**Issue by fperiago** [lululxvi/deepxde#1942]
Hi all,
Up to my best knowledge, binary cross entropy loss function isn't included in deepxde.
Is there a way to include this loss?
Thank you in adavnce

**Comment by fperiago** [comment]
Got it! It was just a problem of resetting the computer.
Sorry!
